jakartaee9/JakartaEE9ReleasePlanFAQ.md

```diff
@@ -34,4 +38,9 @@ Pruned Jakarta EE Projects will leave their property names in the `javax.*` name
 ### Will the target namespace change for schemas?
 
 Yes, the proposed target namespace is `https://jakarta.ee/xml/ns/jakartaee/`.  
-This work is being tracked [via this bug](https://github.com/jakartaee/jakarta.ee/issues/592).
+This work is being tracked [via this bug](https://github.com/jartaee/jakarta.ee/issues/592).
```

Comment:
Looks like a typo was introduced in this line.

Reply:
Thanks, @bshannon!  I should have looked at this closer.  I figured it was marked as changed due to the new line I had to enter...